src/components/MovieDetails/index.tsx

```diff
@@ -111,6 +117,9 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
     return <Error statusCode={404} />;
   }
 
+  const studios = data.productionCompanies;
+  const showAllStudios = studios.length - minStudios <= minStudios;
```

Comment:
```suggestion
  const showAllStudios = studios.length - minStudios < 2;
```
I think it's only necessary to force the display of all studios when the `Show More`/`Show Less` buttons wouldn't actually save any space (i.e., when the button just replaces a single entry). 🤔

We can simply increase the minimum value (5 seems reasonable?) if we want this behavior to trigger less often.

But I'm sure @sct has an option on this, so maybe see that he says about it.

Reply:
Took your suggestion, but tweaked the logic.